TFT/src/User/Menu/common.h

```diff
@@ -52,6 +52,14 @@ extern const uint16_t iconToggle[ITEM_TOGGLE_NUM];
 // Check if next screen update is due
 bool nextScreenUpdate(uint32_t duration);
 
+#ifdef FRIENDLY_Z_OFFSET_LANGUAGE
+  void updateMenuIfInvertedZAxis(MENUITEMS * menuItems);
+
+  #define UPDATE_MENU_IF_INVERTED_Z_AXIS(menuItemsPtr) updateMenuIfInvertedZAxis(menuItemsPtr)
```

Comment:
UPDATE_MENU_IF_INVERTED_Z_AXIS looks like a very long sentence. Replace it with INVERT_Z_AXIS_ICONS to make it concise.